### ISSUE TITLE
API Console Basic Authentication

### ISSUE THREAD

**Issue by yamsellem** [mulesoft/api-workbench#306]
Is there a problem with those security schemes?
In the api console Security Scheme list, only oauth is listed.

![basic](https://cloud.githubusercontent.com/assets/606754/18478241/cce5cc44-79d0-11e6-984c-ea40bde2aff0.png)

```
/resource:
    is: [authenticated]
    get:

securitySchemes:
    basic:
        type: Basic Authentication
    oauth:
        type: OAuth 2.0

traits:
    authenticated:
        securedBy: [basic, oauth]
```

**Comment by ddenisenko** [comment]
Reproduced.

**Comment by dreamflyer** [comment]
will be published in v0.8.39.
